mappings/net/minecraft/GameAndWorldVersion.mapping

```diff
@@ -1,2 +1,4 @@
 CLASS net/minecraft/unmapped/C_xyxorzsk net/minecraft/GameAndWorldVersion
+	METHOD getSeriesId getSeriesName ()Ljava/lang/String;
+	METHOD getWorldVersion getWorldVersionId ()I
```

Comment:
See other comment
```suggestion
	METHOD getWorldVersion getWorldVersion ()I
```

Reply:
I think we should use Id since it returns an int and imo it would be assumed that getWorldVersion would return a string name.
![image](https://user-images.githubusercontent.com/72476111/158027965-38d57644-dd3a-44fd-9104-82c10ad857af.png)